test/compiler.test.js

```diff
@@ -1564,6 +1564,21 @@ describe('compiler', function() {
         .to.throw(/path-does-not-exist/);
     });
 
+    it(
+      'does not fail when an optional middleware cannot be resolved',
+      function() {
```

Comment:
Using just "middleware" instead of "module middleware" does not cross the 80 characters limit. Will update the formatting accordingly.

And thanks for the info on the 80 characters limit.